### PR TITLE
Add a test for uniform buffer restoration across frames.

### PR DESCRIPTION
**Pull request by kainino0x**
Regression test for http://crbug.com/722060 , already passes in Firefox.

Also adds the test from #2388 into 00_test_list.txt.

### CODE REVIEW

File: sdk/tests/conformance2/buffers/00_test_list.txt [hunk 1]
```diff
@@ -7,3 +7,5 @@ buffer-overflow-test.html
 get-buffer-sub-data.html
 one-large-uniform-buffer.html
 uniform-buffers.html
+uniform-buffers-second-compile.html
+uniform-buffers-state-restoration.html
```
Comment (zhenyao) [inline sdk/tests/conformance2/buffers/00_test_list.txt]: Should add --min-version 2.0.1 for book keeping purpose

Comment (kenrussell) [inline sdk/tests/conformance2/buffers/00_test_list.txt]: Could you add --min-version 2.0.1 for both of these just for bookkeeping?

File: sdk/tests/conformance2/buffers/uniform-buffers-state-restoration.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    gl.bindBufferBase(gl.UNIFORM_BUFFER, 0, buf);
+    gl.bufferData(gl.UNIFORM_BUFFER, green, gl.STATIC_DRAW);
+    // Bind throwaray uniform buffer (from only the generic binding point)
+    gl.bindBuffer(gl.UNIFORM_BUFFER, throwawayBuf);
+}
+
+function runFrame() {
+    // ONLY the binding point at index 0 (not the generic binding point) should be correct.
+    // (The generic binding point should point at the throwaway buffer.)
+    if (frame == 9) {
```
Comment (zhenyao) [inline sdk/tests/conformance2/buffers/uniform-buffers-state-restoration.html]: Please use wtu.waitForComposit

**Comment by kainino0x** [comment]
Thanks, done.

**Comment by kenrussell** [comment]
Defer to @zhenyao for the merge since his review was more thorough than mine. :)

**Comment by zhenyao** [comment]
LGTM

**Pull request merged**